server/handlers/common_handlers.go

```diff
@@ -151,3 +152,14 @@ func (h *Handler) DownloadHandler(responseWriter http.ResponseWriter, request *h
 		return
 	}
 }
+
+func GetRefURL(req *http.Request) string {
+	refURL := req.URL.Path + "?" + req.URL.RawQuery
```

Comment:
@MUzairS15 @Jougan-0 please confirm that Meshery supports case-insensitive URL redirection.